frontend/components/FileBrowser/index.tsx

```diff
@@ -9,9 +9,9 @@ import {
 import { ChonkyIconFA } from 'chonky-icon-fontawesome';
 import React, { useMemo, useState } from 'react';
 import { useTranslation } from 'next-i18next';
-import { useTheme } from '@mui/material/styles';
+import { ThemeProvider, useTheme } from '@mui/material/styles';
```

Comment:
Why was this changed? This page is very buggy right now, I wonder if this fixes the issues 🧐